src/actions/sources.js

```diff
@@ -13,8 +13,8 @@ import defer from "../utils/defer";
 import { PROMISE } from "../utils/redux/middleware/promise";
 import assert from "../utils/assert";
 import { updateFrameLocations } from "../utils/pause";
-import { addBreakpoint } from "./breakpoints";
-import { isEnabled } from "devtools-config";
+import { addBreakpoint, disableBreakpoint } from "./breakpoints";
+// import { isEnabled } from "devtools-config";
```

Comment:
Some leftover comments here and on [line 69?](https://github.com/devtools-html/debugger.html/pull/2994/files#diff-695f8e845eb7e7930a1b665650866b59R69)